intake_cesm/core.py

```diff
@@ -43,6 +56,24 @@ def set_collection(self, collection):
         """Set the active collection"""
         self.__init__(collection)
 
+    def build_collections(self):
```

Comment:
Would an alternate implementation be to make this a class method? In that case the API might be something like
```python
col = cesm_metadatastore.build_cesm_metadatastore(**kwarg)
```
whilst preserving the original interface to init. 

It's not obvious to me which is preferable.